docs/current/sdk/python/459708-index.md

```diff
@@ -0,0 +1,69 @@
+---
+slug: /sdk/python
+---
+
+# Dagger Python SDK
+
+<div class="status-badge">Technical Preview</div>
+
+## What is the Dagger Python SDK?
+
+The Dagger Python SDK contains everything you need to develop CI/CD pipelines in Python, and run them on any OCI-compatible container runtime.
+
+Currently the Python SDK consists of:
+
+* A Python package
+* This documentation
+
+## Who is it for?
+
+The Dagger Python SDK may be a good fit if you are...
+
+* A Python developer wishing your CI pipelines were Python code instead of YAML.
+* A developer who needs CI/CD, and is looking for an excuse to learn Python.
+* Your team's "designated devops person", hoping to replace a pile of artisanal scripts with something more powerful.
+* A platform engineer writing custom Python tooling, with the goal of unifying continuous delivery across organizational silos.
+* A cloud-native developer advocate or solutions engineer, looking to demonstrate a complex integration on short notice.
+
+The Dagger Python SDK may *not* be a good fit if you are...
+
+* A developer who doesn't know Python, and is not interested in learning it.
+* A "designated devops person" who doesn't think of themselves as a developer - nothing makes you happier than a battle-hardened shell script.
```

Comment:
@shykes has feedback on this line, as Python isn't normally the **go** to for devops.

Reply:
Removed